docs/eventing/samples/kubernetes-event-source/README.md

```diff
@@ -151,7 +151,7 @@ spec:
       containers:
         - # This corresponds to
           # https://github.com/knative/eventing-contrib/blob/release-0.5/cmd/event_display/main.go
```

Comment:
@vaikas-google should these "blog/release-0.5" references just all default to master?
```suggestion
          # https://github.com/knative/eventing-contrib/tree/master/cmd/event_display/main.go
```

Reply:
we also have the option of inserting a dynamic variable [`{{% branch %}}`](https://github.com/knative/website/blob/master/layouts/shortcodes/version.md), which would populate this value during knative.dev site build with the corresponding version (ie, will show "release-0.9" in the site). [examples](https://knative.dev/docs/smoketest/#install-version-numbers-and-clone-branch-commands)

Reply:
Done